spec/controllers/admin/conferences_controller_spec.rb

```diff
@@ -280,6 +292,18 @@
         end
       end
     end
+
+    describe 'GET #conference_wide_screen' do
+      it 'requires organizer privileges' do
```

Comment:
@ChrisBr  I think we don't  have to write this at all since cancancan takes cares of  authorization. what do you say?